elements/nuxeo-document-actions/nuxeo-document-parent-inspector-button.js

```diff
@@ -0,0 +1,222 @@
+          <paper-icon-button icon="[[icon]]" noink aria-labelledby="label"></paper-icon-button>
+          <nuxeo-tooltip>[[i18n('parentInspectorButton.tooltip')]]</nuxeo-tooltip>
+        </div>
+      </template>
+
+      <nuxeo-dialog id="dialog" with-backdrop modal>
+        <div class="container">
+          <div class="parentscroll">
+            <div class="scrollable">
+              <div>
+                <h1>Parent Inspector</h1>
+                <div>
+                  <p>This is the parent inspector. It displays information about the parent of this document.</p>
+                  <table class="table">
+                    <tr>
+                      <td>Title:</td>
+                      <td>[[document.contextParameters.firstAccessibleAncestor.title]]</td>
+                    </tr>
+                    <tr>
+                      <td>Path:</td>
+                      <td>[[document.contextParameters.firstAccessibleAncestor.path]]</td>
+                    </tr>
+                    <template is="dom-if" if="[[currentUser.isAdministrator]]">
+                      <tr>
+                        <td>UID:</td>
+                        <td>[[document.contextParameters.firstAccessibleAncestor.uid]]</td>
+                      </tr>
+                    </template>
+                  </table>
+                </div>
+              </div>
+              <div class="facetscontainer">
+                <h1>Facets:</h1>
+                <div class="facets">
+                  <template is="dom-repeat" items="[[_facets]]" as="facets">
+                    <div class="show-items">[[facets]]</div>
+                  </template>
+                </div>
+              </div>
+
+              <template is="dom-if" if="[[currentUser.isAdministrator]]">
+                <div class="schemascontainer">
+                  <h1>Schemas:</h1>
+                  <div class="schemas">
+                    <template is="dom-repeat" items="[[_schemas]]" as="schemas">
+                      <div class="show-items">[[schemas.prefix]]:[[schemas.name]]</div>
+                    </template>
+                  </div>
+                </div>
+              </template>
+            </div>
+            <div class="actions">
+              <paper-button noink class="secondary" on-tap="_closeDialog" role="button" tabindex="0"
+                >[[i18n('command.close')]]</paper-button
+              >
+            </div>
+          </div>
+        </div>
+      </nuxeo-dialog>
+    `;
+  }
+
+  static get is() {
+    return 'nuxeo-document-parent-inspector-button';
+  }
+
+  static get properties() {
+    return {
+      document: {
+        type: Object,
+        notify: true,
+      },
+      icon: {
+        type: String,
+        value: 'icons:info',
+      },
+      _facets: {
+        type: Array,
+        value: [],
+      },
+      _schemas: {
+        type: Array,
+        value: [],
+      },
+    };
+  }
+
+  _isAvailable(docs) {
+    return docs && true;
```

Comment:
`docs && true` is the same as just `docs`, right? If you want it to be transformed into a boolean, you can do something like this:
```suggestion
    return !!docs;
```